libafl/src/stages/dump.rs

```diff
@@ -52,14 +52,15 @@ where
     Z: UsesState,
     Z::State: HasCorpus + HasSolutions + HasRand + HasMetadata,
 {
+    type Status = (); // if this fails, we have bigger problems
```

Comment:
Maybe assert that this didn't fail?

Reply:
The assertion is implied by a crash of the fuzzer :joy: The only way it would fail is if an unhandle-able error occured.